config.go

```diff
@@ -106,6 +108,27 @@ func (c *Config) Discover() error {
 		}
```

Comment:
In the above code - I believe the lookup order is actually backwards from what we want - PWD should override builtin. Had a dive into this semi-recently with a user who wanted to build a branch with newer AWS resources and use it on atlas.

Not saying you have to fix that in the same go - just mentioning here that we should get this fixed too mostly for my own remembering. :grinning: